### PR TITLE
{bio}[foss/2022b] SUPPA v2.3-20231005

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4008.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 535.104.12, Python 3.6.8
See https://gist.github.com/PetrKralCZ/35f497433872986578a01e0ada17a678 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/s/SUPPA/SUPPA-2.3-20231005-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+checksums = ['5e55911be2727c7abc7a4e3b6179f2f7cb0e3126bb443bce102fa7af97249e5d']
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('statsmodels', '0.14.0'),
+    ('scikit-learn', '1.2.1'),
+]
+
+postinstallcmds = ['cd %(installdir)s && mkdir bin && ln -s suppa.py bin/suppa.py && chmod a+x suppa.py']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SUPPA/SUPPA-2.3-20231005-foss-2022b.eb:25]: symlink is not correct here, the `suppa.py` in `bin` should point to `../suppa.py`, not `suppa.py`

if you create the symlink correctly, please check for `bin/suppa.py` in `sanity_check_files`, and remove the `PATH` update in `modextrapaths` (no longer needed, EB will automatically add `bin` to `$PATH`)

Comment (smoors) [reply easybuild/easyconfigs/s/SUPPA/SUPPA-2.3-20231005-foss-2022b.eb:25]: or just remove the symlink, it's not worth it imo.

Comment (smoors) [reply easybuild/easyconfigs/s/SUPPA/SUPPA-2.3-20231005-foss-2022b.eb:25]: (to fix the symlink, add option `-r` to `ln`)

Comment (PetrKralCZ) [reply easybuild/easyconfigs/s/SUPPA/SUPPA-2.3-20231005-foss-2022b.eb:25]: fixed in new PR https://github.com/easybuilders/easybuild-easyconfigs/pull/19275

**Pull request merged**